lib/stat_tracker.rb

```diff
@@ -119,13 +118,18 @@ def games_won_per_team_for(season_id)
     end
   end
 
-  def shots_to_goals_ratio_per_team
-    total_shots_per_team.merge(total_goals_per_team){|team_id, shots, goals| (shots.to_f / goals).round(2)}
+  def shots_to_goals_ratio_per_team(season_id)
+    total_shots_per_team_per_season(season_id).merge(total_goals_per_team_for_season(season_id)){|team_id, shots, goals| (shots.to_f / goals).round(2)}
   end
```

Comment:
I like how we've been ending these methods with "for_season(season_id)" maybe you could do that here.

Reply:
And done!